ringling/param_sets.py

```diff
@@ -0,0 +1,110 @@
+"""
+Copyright 2023 MSOE DISE Project
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import sys
+import pprint
+import requests
+from ringling.response_handling import handle_create
+from ringling.response_handling import handle_get
+from ringling.response_handling import handle_modify
+
+def get_url(base_url):
+    """
+    Get the URL for interacting with projects
+    :return: The project URL
+    """
+    return base_url + "/v1/parameter_sets"
+
+
+def create_param_set(base_url, project_id, training_params, is_active):
+    """
+    Create a parameter set on the Ringling service
+    :param base_url: The URL of the Ringling Service
+    :param project_id: The ID of the project that the parameter set belongs to
+    :param training_params: The parameters of the parameter set
+    :param is_active: If the parameter set is active or inactive
+    :return: None
+    """
+    obj = {"project_id": project_id,
+           "training_parameters": training_params,
+           "is_active": is_active}
+
+    try:
+        response = requests.post(get_url(base_url),
+                                 json=obj, timeout=5)
+        if handle_create(response):
+            response_json = response.json()
+            print(f"Parameter Set created successfully with ID {response_json['parameter_set_id']}")
+        else:
+            sys.exit(1)
+    except requests.exceptions.ConnectionError:
+        print("Can not connect to model management service. Is Ringling running?", file=sys.stderr)
+        sys.exit(1)
+
+def get_param_set(base_url, param_set_id):
+    """
+
+    :param base_url: The URL of the Ringling Service
+    :param param_set_id: The ID of the parameter set
+    :return: None
+    """
+    url = get_url(base_url) + "/" + str(param_set_id)
+    try:
+        response = requests.get(url, timeout=5)
+        if handle_get(response, "Parameter Set", param_set_id):
+            response_json = response.json()
+            pprint.pprint(response_json)
+        else:
+            sys.exit(1)
+    except requests.exceptions.ConnectionError:
```

Comment:
Just import this (e.g., `from requests.exceptions import ConnectionError`)

Reply:
Looks like pylint isn't too happy about this (ringling/param_sets.py:20:0: W0622: Redefining built-in 'ConnectionError' (redefined-builtin))

Reply:
What about doing something like this?

```
from requests.exceptions import ConnectionError as RequestsConnectionError

try:
    ...
exception RequestsConnectionError as e:
    ...
```